content/en/docs/setup/install/istioctl/index.md

```diff
@@ -270,16 +270,19 @@ $ istioctl manifest apply -f samples/operator/pilot-k8s.yaml
 {{< /text >}}
 
 {{< tip >}}
-For backwards compatibility, the previous [Helm installation options](/docs/reference/config/installation-options/)
-are also fully supported. To set them on the command line, prepend the option name with "`values.`".
+For backwards compatibility, most of the previous [Helm installation options](/docs/reference/config/installation-options/)
+are also supported except for kubernetes resources. To set them on the command line, prepend the option name with "`values.`".
 For example, the following command overrides the `pilot.traceSampling` Helm configuration option:
 
 {{< text bash >}}
 $ istioctl manifest apply --set values.pilot.traceSampling=0.1
 {{< /text >}}
 
-Helm values can also be set in an `IstioOperator` definition as described in
+Helm values can also be set in an `IstioOperator` YAML file as described in
 [Customize Istio settings using the Helm API](#customize-istio-settings-using-the-helm-api), below.
+
+If you want to set kubernetes resource settings, it is recommended to use `IstioOperator` API as described in
```

Comment:
s/it is recommended to/you must/ ?
the fact is, using helm api just doesn't work.

Reply:
```suggestion
If you want to set Kubernetes resource settings, use the `IstioOperator` API as described in
```

Reply:
done